spec/takeaway_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'takeaway'
+
+describe Takeaway do
+  let(:menu) { double :menu }
+  let(:shop) { described_class.new(menu) }
+  let(:name_one) { "name one" }
+  let(:price_one) { "£0.01" }
+  let(:name_two) { "name two" }
+  let(:price_two) { "£0.02" }
+  let(:item_one) { double :item, :name => name_one, :price => price_one, :available => true }
+  let(:item_two) { double :item, :name => name_two, :price => price_two, :avaialble => true }
+
+  
+  describe '#menu' do
+    it "can be accessed" do
+      expect(shop).to respond_to(:menu)
+    end
```

Comment:
Remove these - respond_to matchers are only useful during code writing.. and even then I'd skip them.